postprocessing/janus-pp-rec.c

```diff
@@ -255,12 +257,18 @@ int main(int argc, char *argv[])
 	}
 
 	/* If we're asked to print the JSON header as it is, we must not print anything else */
-	gboolean jsonheader_only = FALSE, header_only = FALSE, parse_only = FALSE;
+	gboolean jsonheader_only = FALSE, header_only = FALSE, ext_only = FALSE, parse_only = FALSE;
+	janus_pp_extension_report *report = NULL;
+
 	if(args_info.json_given)
 		jsonheader_only = TRUE;
 	if(args_info.header_given && !jsonheader_only)
 		header_only = TRUE;
-	if(args_info.parse_given && !jsonheader_only && !header_only)
+	if(args_info.ext_given) {
+		ext_only = TRUE;
+		report = janus_pp_create_ext_report();
```

Comment:
Not sure why you're creating this here, considering it will only be used much later. Besides, there are points in the code where we might exit sooner (e.g., in case of errors), so this would leak.